java/test/jmri/jmrix/loconet/LnOpsModeProgrammerTest.java

```diff
@@ -524,8 +524,8 @@ public void testOpsReadLocoNetModeLACKNotImplemented() throws ProgrammerExceptio
         Assert.assertEquals("still one message sent", 1, lnis.outbound.size());
         Assert.assertEquals("No programming reply", 0, pl.getRcvdInvoked());
 
-        // LACK "function not implemented" followed by Known-good message in reply
-        m = new LocoNetMessage(new int[]{0xB4, 0x6F, 0x7F, 0x5B});
```

Comment:
Wouldn't it be good to test this case too? But maybe it doesn't matter?

Reply:
Based on the LocoNet traces I've gotten (and a new reading of the PE spec) LACK 0x7F by itself shouldn't be happening.  If it does, JMRI will just ignore it  and wait for another one. So there's no much to test here that's not convered by the new SlotManager tests.